azure-devops/azext_devops/dev/team/_format.py

```diff
@@ -6,6 +6,32 @@
 from collections import OrderedDict
 
 
+def transform_extension_table_output(result):
+    table_output = [_transform_extension_row(result)]
+    return table_output
+
+
+def transform_extensions_table_output(result):
+    table_output = []
+    for item in sorted(result, key=_get_extension_key):
+        table_output.append(_transform_extension_row(item))
+    return table_output
+
+
+def _transform_extension_row(row):
+    table_row = OrderedDict()
```

Comment:
Seems like lot of columns do these generally fall within character limits on the screen? 
We can trim longer fields or reduce the number of fields to make sure table is not broken.

Reply:
ack